halo/app/start.go

```diff
@@ -95,8 +95,10 @@ func Start(ctx context.Context, cfg Config) (<-chan error, func(context.Context)
 		return nil, nil, err
 	}
 
-	if err := enableSDKTelemetry(); err != nil {
-		return nil, nil, errors.Wrap(err, "enable cosmos-sdk telemetry")
+	if cfg.Network != netconf.Simnet {
```

Comment:
I had to do this in order to not get the test suite to fail when it's run (individual tests are fine). Any better suggestions?